pom.xml

```diff
@@ -3,74 +3,46 @@
+  <version>unversioned</version>
+  <packaging>jar</packaging>
+  <name>docs.developer.ch.gov.uk</name>
+  <description>Static content for the developer hub,
+    such as "getting started" and "introduction" documentation
+  </description>
 
-            <pageNotFound.path>dev-hub/error/404Error</pageNotFound.path>
-          </environmentVariables>
-        </configuration>
-        <dependencies>
-          <dependency>
-            <artifactId>junit-platform-surefire-provider</artifactId>
-            <groupId>org.junit.platform</groupId>
-            <version>${junit-platform-surefire-provider.version}</version>
-          </dependency>
-        </dependencies>
-        <groupId>org.apache.maven.plugins</groupId>
-        <version>${maven-surefire-plugin.version}</version>
-      </plugin>
+  <parent>
+    <artifactId>companies-house-parent</artifactId>
+    <groupId>uk.gov.companieshouse</groupId>
+    <version>1.3.0</version>
+    <relativePath/>
+  </parent>
 
+  <properties>
+    <common-web-java.version>1.1.0-rc1</common-web-java.version>
+    <java-session-handler.version>2.2.0</java-session-handler.version>
+    <java.version>1.8</java.version>
+    <junit-jupiter-engine.version>5.5.2</junit-jupiter-engine.version>
+    <maven.compiler.source>${java.version}</maven.compiler.source>
+    <!--    These versions are earlier than in the Parent POM-->
+    <maven.compiler.target>${java.version}</maven.compiler.target>
+    <mockito-junit-jupiter.version>3.1.0</mockito-junit-jupiter.version>
 
-      <plugin>
-        <artifactId>spring-boot-maven-plugin</artifactId>
-        <version>${spring-boot-maven-plugin.version}</version>
-        <configuration>
-          <layout>ZIP</layout>
-          <mainClass>uk.gov.ch.developer.docs.DocsWebApplication</mainClass>
-        </configuration>
-        <executions>
-          <execution>
-            <goals>
-              <goal>repackage</goal>
-            </goals>
-          </execution>
-        </executions>
-        <groupId>org.springframework.boot</groupId>
-      </plugin>
-      <plugin>
-        <artifactId>jacoco-maven-plugin</artifactId>
-        <version>${jacoco-maven-plugin.version}</version>
-        <groupId>org.jacoco</groupId>
-      </plugin>
-    </plugins>
-  </build>
+    <structured-logging.version>1.5.0-rc2</structured-logging.version>
+    <thymeleaf-extras-java8time.version>3.0.1.RELEASE</thymeleaf-extras-java8time.version>
+    <thymeleaf-layout-dialect.version>2.3.0</thymeleaf-layout-dialect.version>
+    <web-security-java.version>1.3.8</web-security-java.version>
+    <oauth-signin-java.version>0.2.2</oauth-signin-java.version>
+    <spring-boot-dependencies.version>2.2.0.RELEASE</spring-boot-dependencies.version>
+    <spring-test.version>5.1.3.RELEASE</spring-test.version>
+    <spring-boot-maven-plugin.version>2.2.0.RELEASE</spring-boot-maven-plugin.version>
+    <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
```

Comment:
Maybe worth collecting the maven properties together & same with the Spring ones under `<!-- Spring -->`